packages/financial-templates-lib/src/price-feed/TraderMadePriceFeed.js

```diff
@@ -0,0 +1,255 @@
+const { PriceFeedInterface } = require("./PriceFeedInterface");
+const { parseFixed } = require("@uma/common");
+
+// An implementation of PriceFeedInterface that uses TraderMade api to retrieve prices.
+class TraderMadePriceFeed extends PriceFeedInterface {
+  /**
+   * @notice Constructs the TraderMadePriceFeed.
+   * @param {Object} logger Winston module used to send logs.
+   * @param {Object} web3 Provider from truffle instance to connect to Ethereum network.
+   * @param {String} pair Representation of the pair the price feed is tracking.
+   * @param {String} apiKey TraderMade Data API key.
+   * @param {Integer} lookback How far in the past the historical prices will be available using getHistoricalPrice.
+   * @param {Object} networker Used to send the API requests.
+   * @param {Function} getTime Returns the current time.
+   * @param {Integer} minTimeBetweenUpdates Min number of seconds between updates. If update() is called again before
+   *      this number of seconds has passed, it will be a no-op.
+   * @param {Number} priceFeedDecimals Number of priceFeedDecimals to use to convert price to wei.
+
+   */
+  constructor(logger, web3, apiKey, pair, lookback, networker, getTime, minTimeBetweenUpdates, priceFeedDecimals = 6) {
```

Comment:
As mentioned above, I think its preferred if the default `priceFeedDecimals` is 18